src/runtime/auto-prompt-manager.ts

```diff
@@ -516,11 +516,14 @@ export class AutoPromptManager {
     if (this.isValidFrequencyCapDuration_(globalFrequencyCapDuration)) {
       const globalTimestamps = Array.prototype.concat.apply(
         [],
-        Object.entries(actionsTimestamps!).map(([type, timestamps]) =>
-          type === nextOrchestration!.type
-            ? timestamps.completions // Completed repeatable actions count towards global frequency
-            : timestamps.impressions
-        )
+        Object.entries(actionsTimestamps!)
+          // Ignore events keyed by configId before FCA Phase 1 rampup
+          .filter(([key, _]) => key in InterventionType)
```

Comment:
I believe the `in` operator will check against the keys of InterventionType. That's not necessarily a problem since the keys and values are the same, but I think we use the values as authoritative and I worry that this could cause a mismatch in the future. May not be a concern if we're going to remove this after the rampup.